independent-publisher-connectors/Edatalia sign online/apiDefinition.swagger.json

```diff
@@ -0,0 +1,625 @@
+                                },
+                                "height": {
+                                  "type": "integer",
+                                  "format": "int32",
+                                  "description": "Widget height",
+                                  "title": "Height"
+                                }
+                              },
+                              "description": "widget"
+                            }
+                          }
+                        },
+                        "description": "widgets"
+                      }
+                    }
+                  },
+                  "description": "recipients"
+                },
+                "documents": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "id": {
+                        "type": "string",
+                        "description": "Unique name for the document",
+                        "title": "Document id"
+                      },
+                      "name": {
+                        "type": "string",
+                        "description": "Document name",
+                        "title": "Document name"
+                      },
+                      "b64PDFContent": {
+                        "type": "string",
+                        "description": "PDF document in base 64",
+                        "title": "Base64 PDF Content"
+                      }
+                    }
+                  },
+                  "description": "documents"
+                }
+              },
+              "required": [
+                "sendMethod"
+              ]
+            }
+          }
+        ]
+      }
+    },
+    "/PSC/v40/DocumentSet/document/{documentSetId}/{documentId}": {
+      "get": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
+          }
+        },
+        "summary": "Get signed document (multiple)",
+        "description": "Get signed document for multiple documents signed",
+        "operationId": "Get-signed-document-multiple",
+        "parameters": [
+          {
+            "name": "documentSetId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Document set unique identifier",
+            "x-ms-summary": "Document Set Id",
+            "x-ms-url-encoding": "single"
+          },
+          {
+            "name": "documentId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Unique identifier for the document signed",
+            "x-ms-summary": "Document Id",
+            "x-ms-url-encoding": "single"
+          }
+        ]
+      }
+    },
+    "/PSC/v40/DocumentSet/{documentSetId}": {
+      "get": {
+        "responses": {
+          "default": {
```

Comment:
@marpedroche default responses are not allowed. Please define a proper response. please see our docs [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/coding-standards#responses) and [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#operations)
Please fix all responses in the swagger file